.github/workflows/default.yml

```diff
@@ -68,3 +76,37 @@ jobs:
           build-root-directory: test-project
           gradle-version: ${{ matrix.gradle }}
           arguments: formatKotlin lintKotlin
+
+  integration-tests-android:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        gradle: [ current ]
+        java: [ 17 ]
+        agp: [ 7.0.4 ]
+        include:
+          - gradle: 7.0.1
+            java: 11
+            agp: 4.2.2
+          - gradle: 6.9.1
+            java: 8
+            agp: 4.0.0
```

Comment:
This version isn't found. Perhaps it requires a different repo or a different patch version?
I'm fine dropping that from the matrix if it end up being a pain.

Reply:
Yeah, I wasn't sure what's the lower boundary so I picked a version I consider absolute minimum and I would be surprised if any continously maintained project used lower version. I know some project refrain from migrating to 7.0 because it still has some hickups and I _heard_ the 7.2 should be considered the "proper" version of AGP 7 (7.2 is still in alpha and is planned for mid February) 

My initial setup didn't work because older AGP versions don't publish gradle metadata needed for `plugins` block ([Issuetracker](https://issuetracker.google.com/issues/64551265)). I dropped `plugins`  block in favor of custom `resolutionStrategy`[here](https://github.com/jeremymailen/kotlinter-gradle/pull/229/files#diff-124a21ab4c45e752a2d3151e279332a337c772e9198fdd248dd43a1127c3e6c9R8)

>I'm fine dropping that from the matrix if it end up being a pain.

It wasn't that big pain so I decided to include necessary changes. My motivation was to explicitly support 4.0 to see the workflow failing when some changes in Kotlinter plugin will introduce breaking changes, so it can be mentioned in the release notes